aten/src/ATen/core/ATenDispatch.h

```diff
@@ -0,0 +1,1263 @@
+#pragma once
+
+// @generated by aten/src/ATen/gen.py
+
+#include <c10/core/Backend.h>
+#include <unordered_map>
+
+// This dispatch class serves as a replacement for Type based dispatch.
+// It stores a function for each native function and backend pair.
+// Functions that are the same for all backends can be registered under
+// Backend::Undefined.
```

Comment:
can't you handle it at the write path? I.e. in registerOp just go and populate every entry (but skipping override)? that way can avoid logic (and mutation!) in getOp